e2e/fixtures/docker/Dockerfile

```diff
@@ -1,5 +1,8 @@
 // @OctoLinkerResolve(https://hub.docker.com/_/php/)
 FROM php:php:5.6-apache
 
+// @OctoLinkerResolve(https://hub.docker.com/_/ubuntu/)
+FROM        ubuntu:16.04 AS base
```

Comment:
Even I like to see more and more E2E tests, I think we should not test everything end to end. Mainly because they are super slow. 

A rule of thumb could be, one E2E test per feature and unit tests for variations like this. 

rule of thumb would be interested in your thoughts on this?

Reply:
I agree, the E2E tests take a long time to run already so it makes sense to not test everything especially if we start to add fixtures for the other plugins.